### ISSUE TITLE
Motion detail: create accessibility indicator for when you are about to 'see' motion-text

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#6137]
**Current behavior**
If you navigate inside of a motion detail you have no verbal (and visual also) indicator for when you are about the read the text
it's sth like motion-title > submitter > some-meta-info > motion-text > "reason" > reson-text

**Wanted behavior**
Only for screenreaders should be an indicator for when you are about to read the text 
Like "Motion text" 

It is important that this indicator is not visual
